app/scripts/metamask-controller.js

```diff
@@ -2192,9 +2189,17 @@ export default class MetamaskController extends EventEmitter {
     this.extension.runtime.onMessageExternal.addListener(onMessageReceived);
     // Fire a ping message to check if other extensions are running
     checkForMultipleVersionsRunning();
+
+    if (this.onboardingController.store.getState().completedOnboarding) {
+      this.networkController.lookupNetwork();
+    }
   }
 
-  triggerNetworkrequests() {
+  async triggerNetworkrequests() {
+    // TODO: Delete when ready to remove `networkVersion` from provider object
+    this.deprecatedNetworkId = null;
```

Comment:
Should these two lines be added right after each of the two `this.networkController.lookupNetwork();` calls?